process-engine/process_engine_meta
# Enhance Integration tests for the error end event

## What did you change?
This PR extends the test process for the ErrorEndEvent with boundary events, that tries to catch an unrelated and an anonymous error. 

Both cases should never occur, since the ErrorEndEvent should throw a specified error.

## How can others test the changes?
This PR can only be testes, if the handlers for the ErrorEndEvent are merged. 
See https://github.com/process-engine/process_engine/pull/102

## PR-Checklist

Please check the boxes in this list after submitting your PR:
